cmake/thirdparty/SetupHIP.cmake

```diff
@@ -3,14 +3,11 @@
 #
 # SPDX-License-Identifier: (BSD-3-Clause)
 
-# Author: Noel Chalmers @ Advanced Micro Devices, Inc.
```

Comment:
This no longer bears any relevance to what he originally wrote. This should have been removed when @davidbeckingsale completely rewrote this logic.